site/src/lib/api/model/block.model.ts

```diff
@@ -0,0 +1,193 @@
+import { Db, ObjectId, WithId } from "mongodb";
```

Comment:
This file will change in a subsequent PR, it is mostly stubs at the moment, save what it does to serve the existing local blocks.